File: documentation/user/en/get-started/example/list-entities.java
return evita.queryCatalog(
	"evita",
	session -> {
		return session.queryListOfSealedEntities(
			query(
				collection("Product"),
				filterBy(
					entityLocaleEquals(Locale.ENGLISH)
				),
				require(entityFetchAll())
			)
		);
	}
);

File: documentation/user/en/get-started/example/list-entities.cs
return evita.QueryCatalog(
	"evita",
	session => {
		return session.QueryListOfSealedEntities(
			Query(
				Collection("Product"),
				FilterBy(
					EntityLocaleEquals(new CultureInfo("en"))
				),
				Require(EntityFetchAll())
			)
		);
	} 
);